### PR TITLE
08 SchedulePlanController, SharedModuleMetadataController, SharedModuleController

### PR DESCRIPTION
**Pull request by alxdarksage**
Discovered that mostrecent and includeDeleted differ in case (probably should have picked one approach). They are supported exactly as they are supported in BridgePF (with different casing).

### CODE REVIEW

File: src/main/java/org/sagebionetworks/bridge/spring/controllers/SchedulePlanController.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+@CrossOrigin
+@RestController
+public class SchedulePlanController extends BaseController {
+
+    static final StatusMessage DELETE_MSG = new StatusMessage("Schedule plan deleted.");
+    private SchedulePlanService schedulePlanService;
+    
+    @Autowired
+    void setSchedulePlanService(SchedulePlanService schedulePlanService) {
```
Comment (DwayneJengSage) [inline src/main/java/org/sagebionetworks/bridge/spring/controllers/SchedulePlanController.java:42]: Minor: final

File: src/main/java/org/sagebionetworks/bridge/spring/controllers/SharedModuleController.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+@CrossOrigin
+@RestController
+public class SharedModuleController extends BaseController {
+    
+    private SharedModuleService moduleService;
+
+    /** Shared Module Service, configured by Spring. */
+    @Autowired
+    void setModuleService(SharedModuleService moduleService) {
```
Comment (DwayneJengSage) [inline src/main/java/org/sagebionetworks/bridge/spring/controllers/SharedModuleController.java]: Minor: final

**Review by DwayneJengSage — APPROVED** [review]
Looks good. Minor comments about setting Spring setters as final.

File: src/main/java/org/sagebionetworks/bridge/spring/controllers/SharedModuleController.java (hunk 2 above)
Comment (alxdarksage) [reply src/main/java/org/sagebionetworks/bridge/spring/controllers/SharedModuleController.java]: Updated

**Pull request merged**